example/markers/app.js

```diff
@@ -60,6 +61,10 @@ document.addEventListener('DOMContentLoaded', function() {
         console.log("marker drop", marker.label);
     });
 
+    wavesurfer.on('marker-contextmenu', function(marker) {
+        console.log("marker context menu", marker.label);
```

Comment:
`"marker context menu:"`. If you could add a `:` for the other console logs as well, great (eg. `console.log("marker drop:", marker.label);`)